build-system/tasks/storybook/index.js

```diff
@@ -16,23 +16,29 @@
 'use strict';
 
 const argv = require('minimist')(process.argv.slice(2));
-const {exec} = require('../../common/exec');
+const log = require('fancy-log');
+const {createCtrlcHandler} = require('../../common/ctrlcHandler');
+const {createServer} = require('net');
+const {cyan, yellow} = require('ansi-colors');
+const {DEFAULT_PORT: AMP_BUILD_PORT} = require('../serve');
+const {defaultTask} = require('../default-task');
+const {execScriptAsync} = require('../../common/exec');
 const {installPackages} = require('../../common/utils');
 
-const DEFAULT_PORTS = {
-  'amp': 9001,
-  'preact': 9002,
+const MODE_PORTS = {
+  amp: 9001,
+  preact: 9002,
 };
 
 function runStorybook(mode) {
   // install storybook-specific modules
   installPackages(__dirname);
 
-  const port = argv.port || DEFAULT_PORTS[mode];
+  const {ci, 'storybook_port': port = MODE_PORTS[mode]} = argv;
```

Comment:
If you're not using quotes for `ci` then probably no need for `storybook_port` either?

Reply:
I suspect this is because the linter will complain about the variable name formatting without quotes

Reply:
Yup.